packages/osc-ecommerce/app/components/CookieBanner/CookieBanner.scss

```
@@ -0,0 +1,27 @@
+/* stylelint-disable plugin/selector-bem-pattern */
+@import "../../../../osc-ui/src/styles/settings";
+@import "../../../../osc-ui/src/styles/tools";
+
+@layer components {
+    .c-cookie-banner {
+        position: fixed;
+        bottom: 1%;
+        z-index: 10;
+        width: 375px;
```

Worth ensuring in such cases where a user is on a smaller than usual device - consider max-width so as the banner is not cut off.
```suggestion
        max-width: 375px;
        width: 100%;
```

---

Good shout. Added.